### DISCUSSION TITLE
Folding Lists

### DISCUSSION THREAD

**Discussion by haveanicedavid** [outline/rich-markdown-editor#526] [Ideas]
Hi! Love the project, and really enjoy the recently added folding headers.

i'm curious if there would be any issue adding the same functionality to lists? If it's a feature you'd want, i'm happy to take a whack at it, though looking at the code I'm not sure whether I could take a similar approach to how the header folding works. 

I guess my questions are
1. Is this possible?
2. Is it a feature you want? And if so
3. Do you have any suggestions for how I might implement? 

It's a minor feature, but makes a big difference in my workflow so i'd love to add.

Thanks!

**Comment by tommoor** [comment]
Not against the idea, as long as removing the extension allows it to be easily disabled :) 

I think it could be implemented in a very similar fashion, the problem is identifying the lists – with headers we have a slug based on the content so perhaps it could use a similar method to generate a slug from the list content?
